cmd/jsonnet.cpp

```diff
@@ -618,27 +628,29 @@ int main(int argc, const char **argv)
             return EXIT_FAILURE;
         }
 
-        // Read input files.
-        std::string input;
-        if (!read_input(&config, &input)) {
-            jsonnet_destroy(vm);
-            return EXIT_FAILURE;
-        }
-
         // Evaluate input Jsonnet and handle any errors from Jsonnet VM.
         int error;
         char *output;
         switch (config.cmd) {
             case EVAL: {
+                assert(config.inputFiles.size() == 1);
```

Comment:
I could be reading this wrong, but if `inputFiles` is empty the above check you guard with that "ERROR: Already specified" message won't catch a malformed userinput of 0 files.